Tests/Shared/SampleData/MessagingPushError.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+/// Class to represent generic errors related to push notification operations, specifically for testing MessagingPush API.
+public enum MessagingPushError: Error {
+    /// Indicates a failure in the registration process for push notifications.
+    case registrationFailed
```

Comment:
Random error to test `MessagingPush` APIs

Reply:
This class has the comment: 

> /// Class to represent generic errors related to push notification operations, specifically for testing MessagingPush API.

I was expecting a class name that matched that description. Maybe `GenericError` as class name? `MessagingPushError.registrationFailed` sounds specific and sounds like a real error that could happen. If this error case was a real error that could be thrown, I would expect that error class to be in the `Sources/` module instead of `Tests/`. 

This is a non-blocking suggestion.

Reply:
Updated to `GenericError`